django/db/backends/postgresql/operations.py

```diff
@@ -178,7 +205,7 @@ def compose_sql(self, sql, params):
         return mogrify(sql, params, self.connection)
 
     def set_time_zone_sql(self):
-        return "SET TIME ZONE %s"
+        return "select set_config('TimeZone', %s, false)"
```

Comment:
```suggestion
        return "SELECT set_config('TimeZone', %s, false)"
```
I aligned the case with the rest of the SQL code.